.github/workflows/commitlint.yml

```diff
@@ -13,7 +13,9 @@ jobs:
         with:
           fetch-depth: 0
           ref: ${{ github.event.pull_request.head.sha }}
+          path: pr_nvim
       - uses: rhysd/action-setup-vim@v1
         with:
           neovim: true
-      - run: nvim --clean -es +"lua require('scripts.lintcommit').main({trace=true})"
+      - run: wget https://github.com/neovim/neovim/blob/master/scripts/lintcommit.lua
```

Comment:
This is `master`, not a specific commit?

Reply:
last commit that touched the file is
https://raw.githubusercontent.com/neovim/neovim/caa6992a1071a2ac373bec21085685da4a1790d6/scripts/lintcommit.lua

(and I do think you want the `raw` file here?)

Reply:
I could go either way on this.  I think it's more likely that we forget to update the commit hash and didn't see a specific downside to this.

Reply:
I just mentioned it because it says "known good version", which I read as specific commit. I agree that pulling master is not a huge security risk here.

I still think you want `raw.githubusercontent.com`, unless you want to run the HTML source for the Github webpage ;)